docs/reference/target-frameworks.md

```diff
@@ -288,22 +288,5 @@ Additional frameworks defined by third parties provide compatibility with other
  | | WindowsPhone 8.1 (UWP) |
  | | WindowsPhone 8.0 (SL) |
 
-Additionally, NuGet packages targeting Xamarin can use additional Xamarin-defined frameworks. See [Creating NuGet packages for Xamarin](https://developer.xamarin.com/guides/cross-platform/advanced/nuget/).
```

Comment:
I think this was meant to be an exhaustive list of frameworks and we'd keep it.

For example, the PCL stuff is still in this doc.

Maybe for the time being we can just mark it with a note the same way PCLs are?

Then in a different change we can rethink this doc altogether.

Reply:
I've restored the table.